src/main/java/uk/gov/register/views/representations/spreadsheet/RecordSpreadSheetWriter.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.register.views.representations.spreadsheet;
+
+import uk.gov.register.views.RecordView;
+import uk.gov.register.views.representations.ExtraMediaType;
+import uk.gov.register.views.representations.RepresentationWriter;
+
+import javax.ws.rs.Produces;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.MultivaluedMap;
+import javax.ws.rs.ext.Provider;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+
+@Provider
+@Produces(ExtraMediaType.TEXT_SPREADSHEET)
+public class RecordSpreadSheetWriter extends RepresentationWriter<RecordView> implements RecordSpreadsheet {
```

Comment:
This class is identical to `RecordsSpreadSheetWriter` in all respects except the different in the view that the classes take as a parameter (but the fact that we only use this view to call `getRecords()` means there is no real difference). Did you mean to pass a parameter of `"records"` to the `process` method here, as opposed to `"record"`? If not, do we need both classes?

Reply:
We need both classes because one receives a `RecordsView` and the other one `RecordView`. True that both f them does the same but the signature of the methods `writeTo` are different, and this is the "black magic" done by Dropwizard and the writers. It will be great to be able to pass the parameter `records` and have the behaviour in `RecordSpreadsheet` in one of this classes and delete the other one, but apparently it doesn't work that way.
And, if you observe the turtle writer there are different classes for Records and Record too.

Reply:
Got it. I thought of this point as I saw that `RecordView` is a subclass of `RecordsView`, so wasn't sure by that logic if we would have been able to get away with only one writer.